src/Controller/SessionAPI/BaseController.php

```diff
@@ -2,9 +2,18 @@
 
 namespace App\Controller\SessionAPI;
 
-class BaseController
+use App\Domain\Services\ConnectionManager;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class BaseController extends AbstractController
 {
 
+    public function getSessionEntityManager($sessionId): EntityManagerInterface
+    {
+        $database = ConnectionManager::getInstance()->getGameSessionDbName($sessionId);
+        return $this->container->get("doctrine.orm.{$database}_entity_manager");
```

Comment:
I wonder if the container will always have it ? 
I mean a controller needs to autowire the service if it would need the that entity manager ? 

Or would it be an idea to just have the entity managers auto wired by the connection manager, and that you can just call getSessionEntityManager($sessionId) on it ?